src/main/java/org/broadinstitute/hellbender/utils/read/ReadUtils.java

```diff
@@ -1329,4 +1334,37 @@ public static int getLastAlignedReadPosition(final GATKRead read) {
             }
         }
     }
+
+    /**
+     * This method is used when we go through reads in the likelihood object. Used to be called isUsableRead()
+     **/
+    public static boolean readHasReasonableMQ(final GATKRead read){
+        return read.getMappingQuality() != 0 && read.getMappingQuality() != QualityUtils.MAPPING_QUALITY_UNAVAILABLE;
+    }
+
+    // TODO: duplicate method in OxoGReadCounts.java
+    public static boolean isF2R1(final GATKRead read) {
+        return read.isReverseStrand() == read.isFirstOfPair();
+    }
+
+    /**
+     * Creates a list of standard filters. Override getDefaultFilters() in e.g. LocusWalker and call this method
+     * TODO: ReadFilterLibrary may be a more suitable home for this method
```

Comment:
So I looked into this. I remember David Roazen didn't want HaplotypeCaller and Mutect2 to share read filters, because we don't want a change in one to affect the other. So I looked into extracting common filters among the walkers in the Mutect family - namely, `Mutect2` and `GetPileupSummaries`, and even these two use different sets of filters. I think it's cleanest to actually abandon this helper method and name all filters in each tool

Reply:
Alternatively, `CollectDataForReadOrientaitonFilter` and `GetPileupSummaries` can call `Mutect2Engine.makeStandardMutect2ReadFilters`. It's good enough for `CollectDataForReadOrientaitonFilter`. I leave the decision up to you for `GetPileupSummaries`